src/viz/index.ts

```diff
@@ -29,7 +29,9 @@ export const createFromTemplate = async (
   config.devBucket = addBucketPrefix(config.devBucket);
   config.prodBucket = addBucketPrefix(config.prodBucket);
   const {devBucket, prodBucket, projectName, basePath} = config;
-  const templatePath = path.join(basePath, 'templates', config.projectChoice);
+  const templatePath = config.codelab
```

Comment:
Snice the part of this that's different is just the template name, I would have that be defined through the ternary, not the whole thing. 

```
path.join(basePath, 'templates', newVariableFromTernary)
```